docs/dev/how-to-write-and-run-tests.md

```diff
@@ -55,6 +55,17 @@ make tests
 ```
 
 To run a single test you can use:
+### On Windows:
+
+* for unit test:
+   ```bash
+   make unit_test test="filename.t"
```

Comment:
this will in fact run all unit tests, same for int_test, it will make all integration tests run